provider/gitauth.go

```diff
@@ -16,7 +16,7 @@ func gitAuthDataSource() *schema.Resource {
 		SchemaVersion: 1,
 
 		DeprecationMessage: "Use the `coder_external_auth` data source instead.",
-		Description:        "Use this data source to require users to authenticate with a Git provider prior to workspace creation. This can be used to perform an authenticated `git clone` in startup scripts.",
+		Description:        "~> **Deprecated**\nUse the `coder_external_auth` data source instead.\n\nUse this data source to require users to authenticate with a Git provider prior to workspace creation. This can be used to perform an authenticated `git clone` in startup scripts.",
```

Comment:
What does `~>` do here?

Reply:
This makes it rendered as a warning
https://developer.hashicorp.com/terraform/registry/providers/docs#callouts

Reply:
![image](https://github.com/coder/terraform-provider-coder/assets/10648092/6159eb49-e26e-40f7-863b-598fe184d708)